src/coffee/directives/api/utils/events-helper.coffee

```diff
@@ -6,12 +6,12 @@ angular.module("uiGmapgoogle-maps.directives.api.utils")
         if ignores
           doIgnore = _(ignores).contains(eventName) #ignores to be invoked by internal listeners
         if scope.events.hasOwnProperty(eventName) and angular.isFunction(scope.events[eventName]) and !doIgnore
-          google.maps.event.addListener(gObject, eventName, ->
+          google.maps.event.addListener gObject, eventName, ->
             #$scope.$evalAsync must exist, I have tried null checking, underscore key checking. Nothing works but having a real or fake $evalAsync
             #it would be nice to know why
             unless scope.$evalAsync
               scope.$evalAsync = ->
-            scope.$evalAsync(eventHandler.apply(scope, [gObject, eventName, model, arguments])))
+            scope.$evalAsync(eventHandler.apply(scope, [gObject, eventName, model, arguments]))
```

Comment:
yeah, I was just copying eventsHelper to make sure it was the same, but correct.